docs/variants/asus_kgpe_d16/dasharo-compatibility/flashrom_wp_test_specification.md

```diff
@@ -0,0 +1,203 @@
+# Flashrom Write Protection – Test Specification
+
+## Test cases
+
+### BIOS WP enable test
```

Comment:
each test must have its own unique identifier

Reply:
In the test environment, we add a unique identifier: WPR. @PrzemyslawBanasiak13 please add it to test cases